library/pk.c

```diff
@@ -548,6 +548,7 @@ static size_t uecc_eckey_get_bitlen( const void *ctx )
     return( (size_t) ( NUM_ECC_BYTES * 8 ) );
 }
 
+/* This function compares public keys of two keypairs */
```

Comment:
@AndrzejKurek , I didn't find any calls to this function. It seems it can be removed. Or at least we should fix prv-> pub, it is very confusing

Reply:
I'm not sure how to rename these variables.
There are two keypairs - one, I assume, only has a public key, and the second was designated as the one holding a private key, but also has the private one computed from it (I think). 
If I would rename it to `pair1` and `pair2`, we would lose the information which one of these is the private one.